File: R/Leer Json.R
#install.packages("jsonlite")
library(jsonlite)
library(tidyverse)

datos_json <- fromJSON('C:/Users/USER/Desktop/Codigos/R/TelecomX_Data.json')
#glimpse ayuda a ver informacion hacerca del data frame
glimpse(datos_json)

datos_json <- fromJSON('C:/Users/USER/Desktop/Codigos/R/TelecomX_Data.json', flatten = TRUE)
glimpse(datos_json)
print("----------------- DATOS PLANOS USANDO FLATTEN ------------------")
head(datos_json)

File: Python/Leer Json.py
import pandas as pd

ruta = 'C:/Users/USER/Desktop/Codigos/Python/TelecomX_Data.json'
df = pd.read_json(ruta)
print(df.info())

datos_json_flat = pd.json_normalize(pd.read_json(ruta).to_dict(orient='records'))
print('------------------------------------------')
print('----------- DATOS APLANADOS --------------')
print('------------------------------------------')
print(datos_json_flat.info())
print(datos_json_flat.head(5))